models/DataObject/ClassDefinition.php

```diff
@@ -668,64 +505,13 @@ public function generateClassFiles($generateDefinitionFile = true)
 
             $data = '<?php';
             $data .= "\n\n";
-            $data .= $infoDocBlock;
```

Comment:
@dpfaffenbauer Why did you removed the info doc block from the definition files? We have only this files in the versioning and there we can fast see field changes. Can we restore it again?

Reply:
I noticed this, too. It was quite useful to get a quick overview of what fields/types the class/brick/... has, which is otherwise very difficult to grasp.